pageserver/src/http/routes.rs

```diff
@@ -438,14 +438,29 @@ async fn tenant_status(request: Request<Body>) -> Result<Response<Body>, ApiErro
     let index_accessor = remote_index.read().await;
     let has_in_progress_downloads = index_accessor
         .tenant_entry(&tenant_id)
-        .ok_or_else(|| ApiError::NotFound("Tenant not found in remote index".to_string()))?
```

Comment:
I don't think this should return an error here. A newly created tenant may not be attached to a remote index. Got the error whenever I tried to call `tenant_status` in test.

Can you confirm this behaviour @LizardWizzard ?